app/components/app_programme_stats_component.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class AppProgrammeStatsComponent < ViewComponent::Base
+  def initialize(programme:, patients_count:)
+    super
+    @programme = programme
+    @patients_count = patients_count
```

Comment:
For consistency, could we get the patients count in the same way as we do for vaccinations and consent notifications? I quite like the idea of this component not needing any external data and then it handles everything itself.